airflow/providers/amazon/aws/operators/sagemaker.py

```diff
@@ -608,3 +608,26 @@ def _check_if_job_exists(self) -> None:
                 raise AirflowException(
                     f'A SageMaker training job with name {training_job_name} already exists.'
                 )
+
+
+class SageMakerDeleteModelOperator(BaseOperator):
+    """Deletes a SageMaker model.
+
+    This operator returns True if model was present and deleted else return False if Model was not present.
+
+    :param model_name: The name of Sagemaker Model (templated).
```

Comment:
you mentioned it's templated but you didn't specify templated fields

Reply:
Corrected it. Thanks!